bitbots_vision/src/bitbots_vision/vision_modules/ros_utils.py

```diff
@@ -465,14 +467,23 @@ def speak(string, speech_publisher):
     speak_message.text = string
     speech_publisher.publish(speak_message)
 
-def config_param_change(old_config, new_config, params):
+def set_global_params(params):
```

Comment:
I wouldn't call this `global`, because this could cause irritation regarding global variables.

Reply:
Any better name, for a parameter, which is used by all detectors?

Reply:
general parameters?

Reply:
That sound nice 👍